### PR TITLE
Add a .mailmap file

### PR DESCRIPTION
**Pull request by sschuberth**

### CODE REVIEW

**Comment by ttaylorr** [comment]
Thanks for this. I believe we've had other open PRs in the past for adding `.mailmap` files (https://github.com/github/git-lfs/pull/1479), and closed them since the contributors page on GitHub shows the information accurately.

**Comment by sschuberth** [comment]
But it's still handy for `git shortlog` like @dpursehouse also mentioned in that other PR. It does not feel right to me to rely on a GitHub-specific feature here, it should work with plain Git.

**Comment by dpursehouse** [comment]
I agree with @sschuberth regarding relying on github specific features.  I just didn't have time to follow up on #1479 after it was closed/rejected.

**Comment by sschuberth** [comment]
BTW, my `.mailmap` in this PR is more complete :-D

**Comment by dpursehouse** [comment]
Yes, I was just about to comment the same.

File: .mailmap [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Andy Neff <andyneff@users.noreply.github.com> <andrew.neff@visionsystemsinc.com>
+Artem V. Navrotskiy <bozaro@yandex.ru> a.navrotskiy <navrotskiy@corp.mail.ru>
+Artem V. Navrotskiy <bozaro@yandex.ru> <bozaro@users.noreply.github.com>
+Brandon Keepers <bkeepers@github.com> <brandon@opensoul.org>
+David Pursehouse <david.pursehouse@gmail.com> <dpursehouse@collab.net>
```
Comment (dpursehouse) [inline .mailmap]: This should be the other way around.

Comment (sschuberth) [inline .mailmap]: Done.

**Comment by ttaylorr** [comment]
@sschuberth after some more thought, I figure we can merge this is if you want to reopen the PR.

**Comment by sschuberth** [comment]
@ttaylorr Sounds good! And now a stupid question: How can I reopen this PR, as I do not see such a button? Or do I need to file a new PR?

**Pull request closed without merging**